Commands/Lists/GetListItem.cs

```diff
@@ -118,6 +123,7 @@ protected override void ExecuteCmdlet()
             else
             {
 				CamlQuery query = HasCamlQuery() ? new CamlQuery { ViewXml = Query } : CamlQuery.CreateAllItemsQuery();
+                query.FolderServerRelativeUrl = FolderServerRelativeUrl;
```

Comment:
query.FolderServerRelativeUrl is null by default, so always setting it to the FolderServerRelativeUrl parameter even when FolderServerRelativeUrl is null doesn't have any impact (which means we don't have to null-check it)